UI/GameInfoCache.cpp

```diff
@@ -720,6 +720,7 @@ std::shared_ptr<GameInfo> GameInfoCache::GetInfo(Draw::DrawContext *draw, const
 	if (info->IsWorking()) {
 		// Uh oh, it's currently in process.  It could mark pending = false with the wrong wantFlags.
 		// Let's wait it out, then queue.
+		// NOTE: This is bad because we're likely on the UI thread....
```

Comment:
Ah, I probably have ANRs disabled - oops.  I bet if you use remote disc streaming and your PC is not sharing, you could find a way to run into that by trying to view the detail of a remote game in your recent history...

-[Unknown]